app/src/main/java/com/github/se/icebreakrr/ui/sections/AroundYou.kt

```diff
@@ -63,6 +65,28 @@ fun AroundYouScreen(
   val context = LocalContext.current
   val isConnected = profilesViewModel.isConnected.collectAsState()
 
+  // Define constants for magic numbers
+  val defaultGeoPoint = GeoPoint(0.0, 0.0)
+  val searchRadius = 300.0
+  val verticalPadding = 16.dp
+  val horizontalPadding = 8.dp
+  val messageTextSize = 20.sp
+  val messageTextColor = Color(0xFF575757)
```

Comment:
Nitpick: If they are constants, use const val and write the names in uppercase.

Reply:
I believe in the meantime Maximo did a PR fixing the magic numbers in these 2 screens, I'll merge and accept his when he merges with the main. But you are right.